### PR TITLE
docs: update installation guide

### PR DESCRIPTION
**Pull request by hieu-jan**
The purpose of this pull request is to restructure the installation documentation #632 
This pull request also resolved all the conservations in the closed PR #634

### CODE REVIEW

File: docs/docs/install/windows.md [hunk 1]
```diff
@@ -2,76 +2,19 @@
...
-
-![Chat](/img/chat.png)
+## Uninstall Jan
+If you have installed Jan on your Windows computer, either as a User or System installation, you can easily uninstall it by going to the Windows Control Panel. 
+In case you wish to completely remove all user data associated with Jan after uninstallation, you can delete the user data folders located at `%APPDATA%\Jan`. This will return your system to its state prior to the installation of Jan. This method can also be used to reset all settings if you are experiencing any issues with Jan.
 
-That's it! Enjoy using Large Language Models (LLMs) with 👋Jan.ai.
\ No newline at end of file
+## Common Questions
+### Microsoft Defender SmartScreen prevented an unrecognized app from starting
```
Comment (hieu-jan) [inline docs/docs/install/windows.md:19]: We can remove this section if #655 is solved.

Comment (hieu-jan) [inline docs/docs/install/windows.md:19]: We can remove this section if issue #555 is solved

**Comment by dan-jan** [comment]
@hieu-jan I'll open a PR to this branch, with some WIP changes. Easier for me to explain by writing.

**Comment by dan-jan** [comment]
@hieu-jan @hahuyhoang411 For Documentation, I recommend you guys pass your current writing through a LLM. I use the following prompt: 

"Please transform the following text into a technical documentation format. The new document should be professional, clear, and concise, providing detailed and organized information. It should maintain a formal tone, avoid colloquial language, and be structured with clear headings and bullet points where necessary. The content should be neutral in sentiment, focusing on delivering factual information in a straightforward manner. If applicable, include code blocks, file paths, or URLs to enhance the technical specificity. 

Here is the text to be transformed: [Insert your text here]"

**Comment by dan-jan** [comment]
@hieu-jan I have done an example of this on the "Installing Jan on Windows" PR here. 

While I'm against step-by-step instructions, I would recommend having an image to break up the monotony (e.g. an image of the Windows installer at the top of the screen)

https://github.com/janhq/jan/pull/666

**Comment by hieu-jan** [comment]
@dan-jan Thanks for your valuable feedback. I will update these docs asap!

**Comment by hiento09** [comment]
@hieu-jan , please rebase main by clicking to `Update Branch` or via console then the feature Preview URL for PR will be applied to this PR
image

**Review by 0xSage — APPROVED** [review]
lgtm!

imo installation/overview can be even shorter. but this is a great start!

**Pull request merged**